packages/astro/src/core/messages.ts

```diff
@@ -57,6 +58,14 @@ export function devStart({
 	return messages.map((msg) => `  ${msg}`).join('\n');
 }
 
+export function prerelease({ currentVersion }: { currentVersion: string }) {
+	const tag = currentVersion.split('-').slice(1).join('-').replace(/\..*$/, '');
+	const badge = bgYellow(black(` ${tag} `));
+	const headline = yellow(`▶ This is a ${badge} prerelease build`);
+	const warning = dim(`  Share feedback at ${underline('https://astro.build/issues')}`)
```

Comment:
super duper nit, only if you like it: 

```
Feedback? https://astro.build/issues
```

Also, this means we need to actually create this URL before deploying :) Feel free to use https://github.com/withastro/astro/issues/new/choose so that you're not blocked on merging this today, and then we can add a vanity URL later.

Reply:
Yep, I like that. 👍🏻 
URL already works! https://astro.build/issues